__tests__/wares/pug.test.ts

```diff
@@ -5,7 +5,9 @@ import { renderFile } from 'pug'
 
 describe('PUG templates tests', () => {
   it('should render pug template', async () => {
-    const { fetch, app } = InitAppAndTest((_req, res) => void res.render('../__tests__/wares/pug.test.pug', null, { renderOptions: { pageTitle: 'Test title', foo: 'bar' } }))
+    const { fetch, app } = InitAppAndTest(
+      (_req, res) => void res.render('../__tests__/fixtures/views/pug.test.pug', null, { renderOptions: { pageTitle: 'Test title', foo: 'bar' } })
```

Comment:
fixture files don't have to have `.test.` in their filename. It's for tests. for fixtures just use `test.pug` or smth like this